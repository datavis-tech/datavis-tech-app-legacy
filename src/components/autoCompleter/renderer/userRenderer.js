import { Label } from 'semantic-ui-react'

export default ({ title }) => <Label content={title} />
