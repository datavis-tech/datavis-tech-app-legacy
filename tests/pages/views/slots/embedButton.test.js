jest.mock('../../../../src/routes', () => ({
  findByName: () => ({
    getAs: ({id}) => `some/vis/path/with/${id}`
  })
}))

import React from 'react'
import renderer from 'react-test-renderer'
import EmbedButton from '../../../../src/pages/views/slots/embedButton'

describe('embed button', () => {

  const id = 'Some id'

  let tree

  it('should be a button with a popup', () => {
    tree = renderer.create(<EmbedButton id={id} />).toJSON()
    expect(tree).toMatchSnapshot()
  })

})
