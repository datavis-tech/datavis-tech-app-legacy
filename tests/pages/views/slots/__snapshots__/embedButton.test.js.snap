// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`embed button should be a button with a popup 1`] = `
<button
  className="ui fluid button"
  disabled={undefined}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  role="button"
  style={
    Object {
      "marginTop": "5px",
    }
  }
  tabIndex={undefined}
>
  Embed
</button>
`;
